### DISCUSSION TITLE
How to reset Password with User Name only

### DISCUSSION THREAD

**Discussion by EswarDolu** [community/community#37331] [Other Feature Feedback, Questions, & Ideas]
Hi Team,

Could you please help us how to recover the GitHub account when we forgot the password. When I created I haven't tagged to any email id, Please let us know is there any way to recover the GitHub account

**Comment by tawfik-s** [comment]
you can read here how to get a new password I think your GitHub is associated with email but you forget it 
- try to find your password at your browser password setting

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/updating-your-github-access-credentials

**Comment by airtower-luna** [comment]
That's something you'll have to [ask GitHub support](https://support.github.com/request) about, us other users can't help there. I'm not sure if they'll be able to help though, confirming you're the account owner without anything other than the account name seems… problematic.
